test/memory/uninitialized_copy.cpp

```diff
@@ -43,6 +43,8 @@ namespace {
 		test(ranges::uninitialized_copy(control.cbegin(), control.cend(), independent.cbegin()));
 		test(ranges::uninitialized_copy(control, independent.begin()));
 		test(ranges::uninitialized_copy(control, independent.cbegin()));
+      test(ranges::uninitialized_copy(control, independent));
```

Comment:
whitespace looks odd (tabs vs spaces?)

Reply:
My editor is set to three spaces, and I manually edit them to tabs; looks like this one slipped through the cracks. I'm looking forward to clang-format supporting concepts.